core/chain-configs/src/client_config.rs

```diff
@@ -29,11 +33,19 @@ pub struct GCConfig {
     /// when cleaning forks during garbage collection.
     #[serde(default = "default_gc_fork_clean_step")]
     pub gc_fork_clean_step: u64,
+
+    /// Number of epochs for which we keep store data.
+    #[serde(default = "default_num_epochs_to_keep_store_data")]
+    num_epochs_to_keep_store_data: u64,
```

Comment:
Also, maybe we should rename this? `num_epochs_to_keep`? Though dunno what others think. @mm-near ?

Reply:
I am open to both. @mm-near  what is your preference here?